plugins/src/ios/OptimoveSDKPlugin.swift

```diff
@@ -34,4 +34,105 @@ import NotificationCenter
         guard let config = OptimoveSDKPlugin.config else { return }
         Optimove.initialize(with: config)
     }
+    
+    @objc(reportEvent:)
+    func reportEvent(command: CDVInvokedUrlCommand) {
+        var name = ""
+        var parameters: [String: Any] = [:]
+        
+        if let n = command.arguments[0] as? String {
```

Comment:
Name is required, I think this should cause an error result from the plugin.

Can you compare with the Android implementation for reference?

Reply:
Yes, you're right.

I think it's probably better that if we guard it we can handle the validation in a better way than the Kumulos SDK (Perhaps there were/are some validations at the JS level, but not sure about this).

Let's see how it's implemented on the Optimove Cordova Android side and align with that.

Reply:
add guard to name